js/ui/html_editor/ui.html_editor.js

```diff
@@ -215,6 +220,12 @@ const HtmlEditor = Editor.inherit({
         return renderContentPromise;
     },
 
+    _pointerMoveHandler: function(e) {
+        if(isIos) {
```

Comment:
Did u test vertical scroll? Should we either prevent it?

Reply:
yes, I tested both vertical and horizontal scrollings